effekt/shared/src/main/scala/effekt/machine/Transformer.scala

```diff
@@ -432,6 +449,15 @@ object Transformer {
   def transform(id: Symbol): String =
     s"${id.name}_${id.id}"
 
+  def isSimpleDataType(tpe: core.ValueType): Boolean = {
```

Comment:
What do you mean by simple data type? That it is not boxed?

Reply:
Yes, exactly. I am open to suggestions for a good name here..

Reply:
@me `requiresBoxing`